src/pages/patientView/PatientViewPage.tsx

```diff
@@ -543,10 +543,18 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
                     </MSKTab>)
                     }
 
+
+                    {  (patientViewPageStore.MDAndersonHeatMapAvailable.isComplete && patientViewPageStore.MDAndersonHeatMapAvailable.result ) &&
```

Comment:
couldn't this just be `patientViewPageStore.MDAndersonHeatMapAvailable.result` now?